File: src/org/labrad/qubits/resources/AnalogBoard.java
package org.labrad.qubits.resources;


public class AnalogBoard extends DacBoard {

  public static AnalogBoard create(String name) {
    AnalogBoard board = new AnalogBoard(name);
    return board;
  }

  public AnalogBoard(String name) {
    super(name);
  }
}

File: QubitServer/src/main/scala/org/labrad/qubits/resources/AnalogBoard.scala
package org.labrad.qubits.resources

object AnalogBoard {
  def create(name: String, buildNumber: String, buildProperties: Map[String, Long]): AnalogBoard = {
    new AnalogBoard(name, buildNumber, buildProperties)
  }
}

class AnalogBoard(name: String, buildNumber: String, buildProperties: Map[String, Long]) extends DacBoard(name, buildNumber, buildProperties)
